drivers/i2s/i2s_ll_stm32.c

```diff
@@ -22,11 +22,7 @@
 #include <zephyr/irq.h>
 LOG_MODULE_REGISTER(i2s_ll_stm32);
 
-/* FIXME change to
- * #if __DCACHE_PRESENT == 1
- * when cache support is added
- */
-#if 0
+#if __DCACHE_PRESENT == 1
```

Comment:
Non blocking, can be done in another PR: 
I think this part is obsolete and should be replaced by `sys_cache_data_invd_range()` and `sys_cache_data_invd_range()`.